packages/buidler-core/src/internal/solidity/resolver.ts

```diff
@@ -214,6 +220,39 @@ export class Resolver {
     );
   }
 
+  private async _resolveBuidlerDependency(
+    globalName: string,
+    libraryName: string
+  ) {
+    const buidlerCoreDir = path.join(__dirname, "..", "..");
+    const packagePath = path.join(buidlerCoreDir, "package.json");
+    const absolutePath = path.join(
+      buidlerCoreDir,
+      path.relative(libraryName, globalName)
+    );
+
+    let packageInfo;
+    try {
+      packageInfo = await fsExtra.readJson(packagePath);
+    } catch (error) {
+      throw new BuidlerError(
+        ERRORS.RESOLVER.LIBRARY_NOT_INSTALLED,
```

Comment:
Not sure about this error. The alternative is to create a new, more specific error for this scenario.

Reply:
I'm not sure if I get this `catch` clause. When would this happen?

Reply:
Apart from this, everything looks fine

Reply:
I see. I think I wouldn't wrap this error though, and just let it bubble to the cli. This would be less confusing for users. It'd look like an internal bug, which it would be.

Reply:
That makes sense. Fixed.